service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationJobFlight.java

```diff
@@ -36,6 +38,11 @@ public RunImputationJobFlight(FlightMap inputParameters, Object beanBag) {
         RunImputationJobFlightMapKeys.WDL_METHOD_NAME,
         JobMapKeys.RESULT_PATH.getKeyName());
 
+    PipelinesEnum pipelinesEnum =
+        PipelinesEnum.valueOf(
+            inputParameters.get(JobMapKeys.PIPELINE_NAME.getKeyName(), String.class));
+    MetricsUtils.incrementPipelineRun(pipelinesEnum);
```

Comment:
is there a way to test this in `RunImputationJobFlightTest` ?

Reply:
good point, added!